content/en/docs/languages/dotnet/logs/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: OpenTelemetry .NET Logs
+linkTitle: Logs
+description: Use OpenTelemetry .NET to collect and export log telemetry data
+weight: 35
+---
+
+This section contains information about using OpenTelemetry .NET for logging.
+
+OpenTelemetry .NET Logs provides integration with popular .NET logging
```

Comment:
nit: There is no integration with anything but `ILogger`, so best to call that out.